### PR TITLE
Djangosql

### PR DESCRIPTION
**Pull request by luciencd**
Fixed file structure,
Added more specific columns to django model of votes, legislation, donations, superpacs, representatives.

Added way of exporting ORM model to python file and binding to local database server.

### CODE REVIEW

**Comment by luciencd** [comment]
Fixed merge conflictos

File: server/run.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python /code/src/manage.py runserver 0.0.0.0:8000
```
Comment (SaswatB) [inline server/run.sh]: This was deleted before since its not used anymore.
Please delete again

File: server/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 FROM python:3.6
 ENV PYTHONUNBUFFERED 1
+
```
Comment (SaswatB) [inline server/Dockerfile]: why was this space added

File: server/src/rest/api/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
```
Comment (SaswatB) [inline server/src/rest/api/migrations/0001_initial.py]: Please don't commit auto generated files

File: server/src/rest/rest/settings.py [hunk 4]
```diff
@@ -26,6 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+
```
Comment (SaswatB) [inline server/src/rest/rest/settings.py]: why was this space added

File: server/src/rest/rest/settings.py [hunk 5]
```diff
@@ -77,8 +78,10 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
+
         'NAME': os.getenv('MYSQL_DATABASE', 'test'),
         'HOST': 'mysql',
+
```
Comment (SaswatB) [inline server/src/rest/rest/settings.py]: why were these spaces added

**Comment by SaswatB** [comment]
The api tables are not populated in the mysql database when I test this with docker.

**Pull request merged**